static/locales/fr/translation.json

```diff
@@ -112,12 +112,12 @@
 			"title": "Dépannage",
 			"restore": "Restaurer les paramètres par défaut",
 			"restore_warning": "Cela réinitialisera TOUS les paramètres avec leurs valeurs par défaut.",
-			"warning": "Non disponible comme hôte dans le jeu!"
+			"warning": "Non disponible en tant qu'hôte dans le jeu!"
 		}
 	},
 	"lobbybrowser": {
-		"header": "Lobbies Publiques",
-		"code": "Code du Lobby",
+		"header": "Parties publiques",
+		"code": "Code de la partie",
```

Comment:
I believe the original is more precise as it is the code of the lobby not the code of the group/party

Reply:
No, it's actually correct: in French we say "partie" to speak about a game.